packages/docs-site/src/content/docs/guides/node-operators/enable-a-proposer.mdx

```diff
@@ -23,9 +23,9 @@ These are the bare minimum required settings along with a functional Taiko node
 
 <Steps>
 
-1. First, you will need to have a prover from <a href="https://dojonode.xyz/tatami" target="_blank">the community marketplace</a> or [run your own SGX prover](/guides/node-operators/enable-a-prover).
+1. First, you will need to [set up a prover](/guides/node-operators/enable-a-prover). Follow the guide, and note down the contract address for your `ProverSet` contract before continuing here.
 
-2. You MUST set `PROVER_ENDPOINTS` in simple-taiko-node `.env` file to the your SGX prover's endpoint or one from the marketplace. If you do not set it and leave it default while not running a prover, **your proposer will not work**.
+2. Set the contract address in the `PROVER_SET` env variable.
```

Comment:
Are we just only directing users to use the `ProverSet` way?

Reply:
yes, please see comments above. thanks

Reply:
[5371321](https://github.com/taikoxyz/taiko-mono/pull/17769/commits/537132139fbf1eaa7466b6576180fe82cf4c3961)